config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:index, :show]
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [Optional] Dear @Trust-007, Your route works well and you did great, A great way to organize your route will be to use something like...
``` ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thanks a lot for the suggestion @TadesseDev I'll definitely apply it